.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-20.04
+    env:
+      NODE_ENV: test
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v2.1.2
+      with:
+        node-version: '14'
+
+    - run: cd .operations && npm install
+    - run: cd .operations && npm run lint
```

Comment:
Should there be a linting issue, will we see it within the code in the GitHub "files" tab?

Reply:
I don't think so, but I can check now by introducing a linting issue

Reply:
It's possible, should work by default given the default eslint behaviour

This extension enriches the DX:

https://github.com/marketplace/actions/lint-action

Reply:
When we add eslint we should the GitHub action you suggested, but right now we're using `markdownlint` which doesn't support it (I haven't found a GitHub Action that support that, but I can keep looking)